src/Utils/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+export function getAllArticles() {
+    return axios.get("https://northcoders-news-board.onrender.com/api/articles")
```

Comment:
It would be nice to see an axios instance created with axios.create (see the docs)